src/integration/pages/frontPage/arabic/__snapshots__/amp.test.js.snap

```diff
@@ -2,6 +2,8 @@
 
 exports[`AMP Front Page A11y I can see the skip to content link 1`] = `"إذهب الى المحتوى"`;
 
+exports[`AMP Front Page A11y I can see the skip to content link 2`] = `"إذهب الى المحتوى"`;
```

Comment:
I've removed them and generated them again, and I keep getting the same output 🤷🏻‍♂️

Reply:
Very strange. Because there shouldn't be multiple skip to content links on any page.

Reply:
I have noticed that this duplication is only happening for the first assertions in the tests contained in `runCommonCrossPlatformTests` [here](https://github.com/bbc/simorgh/blob/fdc0bb7fc211b92dc6f7bada966e5682089415c9/src/integration/common/index.js#L13). Might this be weirdly because these assertions are run twice (for both canonical and amp)?

Reply:
If this can't be fixed in this PR (as it might require restructuring our cross-platform integration tests), can we raise another ticket to investigate/fix it in another PR and merge this in?

Reply:
Sure, don't want to block this from getting merged. V strange though!